src/react-components/room/ChatSidebarContainer.js

```diff
@@ -54,146 +48,6 @@ const chatSidebarMessages = defineMessages({
     defaultMessage: "Text Chat Off"
   }
 });
-
-function shouldCreateNewMessageGroup(messageGroups, newMessage, now) {
-  if (messageGroups.length === 0) {
```

Comment:
would you mind just adding a few comments on where all this code went just so I don't have to search too much! thanks!